web/html/src/branding/css/base/theme.scss

```diff
@@ -1332,6 +1332,8 @@ tr.changed > td {
 
 .badge {
   font-size: 12px;
+  border-radius: 10px;
+  background: $eos-bc-gray-1000;
```

Comment:
We probably need the same fix in the old theme files as well, no?

Reply:
It is working fine in the old theme.